### ISSUE TITLE
Taille des images dans le PDF.

### ISSUE THREAD

**Issue by alcoralcor** [faitmain/faitmain.org#33]
Certaines images sont affichées avec une échelle différente dans le PDF, comparé à la version web. Par exemple, _le schéma théorique d'une CNC_, avant dernière image de l'article sur les CNCs. 

http://faitmain.org/volume-1/cncs.html

Mais d'autres articles sont beaucoup plus touchés. Je n'ai pas l'impression que cela soit causé par la source reST, ni par l'image elle-même.

**Comment by tarekziade** [comment]
je me demande si c'est pas la taille d'impression defini dans l'image

**Comment by alcoralcor** [comment]
Effectivement, il semblerait que le jpeg du schema ne contienne pas de données EXIF, contrairement aux autres jpeg du même article - vérifié avec _imagemagick_. C'est la seule différence que j'ai pu repérer. Étant donné qu'il existe des recommandations de taille d'images (en pixels) pour les rédacteurs, il serait pratique que tu puisses forcer une résolution pour le print et ne pas tenir compte des données stockées dans l'image.

Note : Cependant, dans certains articles, les images sont parfois vraiment trop petites et ne suivent pas les recommandations aux auteurs/rédacteurs.

https://github.com/faitmain/faitmain.org/blob/master/src/ecrire.rst

Il faudra peut-être avertir les auteurs pour corriger cela.
